medusa/server/web/home/handler.py

```diff
@@ -1851,29 +1851,27 @@ def subtitleShow(self, indexername=None, seriesid=None):
         return self.redirect('/home/displayShow?indexername={series_obj.indexer_name}&seriesid={series_obj.series_id}'.format(series_obj=series_obj))
 
     def updateKODI(self, indexername=None, seriesid=None):
-        if seriesid is None:
-            return self._genericMessage('Error', 'Invalid show ID')
-
-        indexer_id = indexer_name_to_id(indexername)
-        series_obj = Show.find_by_id(app.showList, indexer_id, seriesid)
+        series_name = series_obj = None
+        if seriesid:
+            indexer = indexer_name_to_id(indexername)
+            series_obj = Show.find_by_id(app.showList, indexer, seriesid)
+            if series_obj is None:
+                return self._genericMessage('Error', 'Unable to find the specified show')
 
-        if series_obj is None:
-            return self._genericMessage('Error', 'Unable to find the specified show')
-        else:
-            show_name = quote_plus(series_obj.name.encode('utf-8'))
+            series_name = quote_plus(series_obj.name.encode('utf-8'))
 
         if app.KODI_UPDATE_ONLYFIRST:
             host = app.KODI_HOST[0].strip()
         else:
             host = ', '.join(app.KODI_HOST)
 
-        if notifiers.kodi_notifier.update_library(series_name=show_name):
+        if notifiers.kodi_notifier.update_library(series_name=series_name):
             ui.notifications.message('Library update command sent to KODI host(s): {host}'.format(host=host))
         else:
             ui.notifications.error('Unable to contact one or more KODI host(s): {host}'.format(host=host))
 
         if series_obj:
-            return self.redirect('/home/displayShow?indexername={series_obj.indexer_name}&seriesid={series_obj.series_id}'.format(series_obj=series_obj))
+            return self.redirect('/home/displayShow?indexername={0.indexer_name}&seriesid={0.series_id}'.format(series_obj))
```

Comment:
I needed to look at this like 3 times before I noticed it was templating. We've not used this before. Maybe it's a little bit better to use this here: `self.redirect('/home/displayShow?indexername={series_obj.indexer_name}&seriesid={series_obj.series_id}'.format(series_obj=series_obj))`

Reply:
What was the reason you changed it?

Reply:
I thought it looked cleaner. I'll revert.